docs/session-insights/client-dashboard.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-sidebar_label: 'Validations'
+sidebar_label: 'Client Dashboard'
```

Comment:
just to confirm, we're using this term instead of 'Validations"?

Reply:
Not quite - Validations are simply part of the client dashboard, this page needs an update (much like the rest of our docs pages). Whenever we document bot instances/running bots and bot instance histories, those concepts will go under this page as well. I think we have a short blurb on these concepts, but they need to be fleshed out more and given their own sections here. If/when we add enough client features that it no longer makes sense to keep them all in one place, we can break them up into separate pages

Reply:
I renamed the title for this page to  "Viewing Validations" so that it reads more like validations can be found within the client dashboard rather than the Client Dashboard being synonymous with and only with Validations

Reply:
I don't have a strong opinion either way. I'd like to push this out of the current PR though - We're for sure going to be adding more content and updating our tutorials. Once we have more "getting started" content, we can address the home page to figure out where we want to direct new users